HW04/Pavel_Mishchenko/spec/student_spec.rb

```diff
@@ -0,0 +1,239 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+RSpec.describe Student do
```

Comment:
Please, add all the changes from the comments for the previous class to this class as well